_posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: Star Wars Jedi Fallen Order - First Impression
+date: 2019-12-05 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [General]
+image:
+image2:
+author: Valentino Urbano
+---
+
+I'm a huge Star Wars fan so view the whole article under this lens. If you're not a Star Wars fan there isn't really much in this game that has not been done bette before in other games, but it being Star Wars makes it special for us.
+
+## PLUS
+
+- Great graphics
+
+The first Star Wars open world game that has good graphics for the time, but only because we didn't get a single SW open world game in the past 10 years. Thanks EA...
+
+- Great gameplay
+
+The puzzles are fun and you don't need to read a guide to understand them (most of the times). The combat is dynamic, with the right level of difficulty (haven't tried the hardest difficulties yet though).
+
+- Few bugs
+
+I'm used to experience lots of bugs in a newly released game. Usually during the first month they are pretty unbearable in most AAA games. Here I did not find any major issue. What I found were really minor problems that did not affect the enjoyment of the game at all.
+
+- Monetization
+
+Finally an EA game that does not have predatory microtransactions or a subcription. What a breath of fresh air.
+
+## MINUS
+
+- Confusing maps
+
+That's not necessarely a downside, only a personal preference. A lot of people will love them. And I'm sure that will time they will start making more sense.
+
+There is also a map screen which helps, but often the maps are on multiple vertical levels adding to the confusion and making the visual map not really super useful in the beginning.
+
+- Timing puzzles
+
+For a few puzzles (the large majority do not present this problem) you need to time your clicks to be very precise or you will fail it. Even (on my estimation) a 300ms delay will make you fail so this is really bad for people either with disabilities or who wants to enjoy the game without too much stress. This happens even in story mode since it is not related to the combat part of the game but to puzzles.
+
+## Conclusion
```

Comment:
Nothing about story or the characters?

Reply:
Can't really say anything about the story until I finish it. Same for the characters I guess. ..?

Reply:
Well you finished it!